example/README.md

```diff
@@ -0,0 +1,30 @@
+# example AWS S3 Creation
+
+Configuration in this directory creates an example AWS public-read S3 bucket with versioning.
+
+This example outputs user name and secrets for the new credentials.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+Run `terraform destroy` when you want to destroy these resources created.
+
+## Outputs
```

Comment:
Let's not repeat the outputs here again, they are already defined in the main README file.

Reply:
done